turbinia/lib/libcloudforensics.py

```diff
@@ -625,37 +687,39 @@ def get_resource_type(self):
     """
     if not self._data:
       operation = self.get_operation().execute()
-      self._data = self.project.gce_operation(
-          operation, zone=self.zone, block=False)
+      self._data = operation
```

Comment:
yes, I agree this is an old method, I opened and issue for it yesterday, so will change it with the new commit. I will remove the if condition completely and make it just:
self._data = self.get_operation().execute()
return self._data.get(key)

reasons:
1-get request is non mutating and will never create a zone operation object, gce operation will just call _execute_operation and return the passed operation object again since block= false, if block is changed to True the method will return a 404 HttpError: zoneOperations and globalOperation objects are only available for mutating api calls.
2- the if condition will return the wrong key if the value is refreshed after self._data is populated on the first time. ex: the status value of a VM

Reply:
That sounds good to me! Thanks for clarifying.